src/features/user/MFV2/ProfileCard/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { ProfileProps } from '../../../common/types/profile';
+import { Avatar } from '@mui/material';
+import getImageUrl from '../../../../utils/getImageURL';
+import styles from './ProfileCard.module.scss';
+import {
+  DefaultUserProfileImage,
+  SettingsIcon,
+} from '../../../../../public/assets/images/icons/ProfilePageV2Icons';
+import { useLocale, useTranslations } from 'next-intl';
+import ProfileActions from './ProfileCardButton/ProfileActions';
+import Link from 'next/link';
+
+const ProfileCard = ({ userProfile, authenticatedType }: ProfileProps) => {
+  const t = useTranslations('Profile');
+  const isPrivateAccount = authenticatedType === 'private';
+
+  return (
+    <div className={styles.profileCardContainer}>
+      <div className={styles.profileBackground}></div>
+      <div className={styles.profilePicture}>
+        {userProfile?.image ? (
+          <Avatar
+            alt="user Image"
+            src={getImageUrl('profile', 'avatar', userProfile?.image)}
+            className={styles.avatar}
+          />
```

Comment:
1. I noticed in the storybook for ProfileCard, that the avatar displays as 'u' as it falls back to the first letter of the alt text. Should we show the default image in this case? 
2. Alt text should be the user name.
3. styles.avatar isn't defined.